assets/section-main-product.css

```diff
@@ -1167,6 +1167,22 @@ a.product__text {
   display: none;
 }
 
+.product__modal-opener > .loading-overlay__spinner {
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  position: absolute;
+  display: flex;
+  align-items: center;
+  height: 48px;
+  width: 48px;
```

Comment:
Do we use `px` elsewhere for icon sizes? I've searched for this element and it seems to be using `rem` based sizes in other locations.